res/values/strings.xml

```diff
@@ -538,8 +549,8 @@
     <string name="qrshow_join_group_hint">Scan this to join the group \"%1$s\".</string>
     <string name="qrshow_join_contact_title">QR invite code</string>
     <string name="qrshow_join_contact_hint">Scan this to set up a contact with %1$s.</string>
-    <string name="qrshow_join_contact_no_connection_hint">QR code setup requires an internet connection. Please connect to a network before proceeding.</string>
```

Comment:
please no changes the english strings in the same pr - this would force all translations to be invalidated, so fixing typos in the english strings should be handled differently in a separated pr.

Reply:
They are so small that it will not be a problem to be handled again.

Reply:
but there is no need to delete >20 translations just because of a typo fix in the english source. therefore, this pr must only contain adding strings. changes to the english sources must be removed completely.

Reply:
You're right, but if corrections are made with new translations, won't we get something more complete?